index.html

```diff
@@ -1,31 +1,79 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
+    <title>This is a Quiz App</title>
+    <meta name="description" content="This is my first Web App" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta charset="UTF-8" />
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
     <header>
       <h1>Quiz App</h1>
     </header>
-    <div class="flex-container1">
-      <p>What properties flip the axes in flexbox? 
+    <section>
+      <div class="flex-container1">
```

Comment:
Maybe you could find an element, that fits better than div

Reply:
Thanks for the review and suggestions @NilsOt1